src/tools/keyboard_control/keyboard_control/keyboard_control.py

```diff
@@ -0,0 +1,89 @@
+import pygame
+import rclpy
+from rclpy.node import Node
+from std_msgs.msg import Bool
+from navigator_msgs.msg import VehicleControl, PedalPosition, SteeringPosition
+
+
+class KeyboardController(Node):
+
+    def __init__(self):
+        #Initialize the VehicleControl Variables
+        self.throttle = 0.0
+        self.steer = 0.0
+        self.brake = 0.0
+        self.reverse = False
+        self.pressed_keys = set()
+        # Initialize Pygame
+        pygame.init()
+
+        #Set up controller screen
+        screen = pygame.display.set_mode((400, 300))
+
+        #Set up node
+        super().__init__("keyboard_controller")
+        self.command_publisher = self.create_publisher(VehicleControl, "/vehicle/control",1)
+        self.input_timer = self.create_timer(0.03  , self.pygame_loop)
+        self.control_timer = self.create_timer(0.03, self.control_callback)
+        
+    #publishes vehicle information to the vehicle control topic
+    def control_callback(self):
+        control_msg = VehicleControl()
+        control_msg.throttle = self.throttle
+        control_msg.brake = self.brake 
+        control_msg.steer = self.steer
+        control_msg.reverse = self.reverse
+        self.command_publisher.publish(control_msg)
+        #print("Published Message!\n")
+    
+    #handles Pygame controls
+    def handle_controls(self):
+        if pygame.K_w in self.pressed_keys:
+            self.throttle += 0.05
+            print("Going forward! \n")
+        elif pygame.K_s in self.pressed_keys:
+            self.throttle -= 0.05
+            self.brake += 0.05
+            print("Braking! \n")
+        elif pygame.K_a in self.pressed_keys:
+            self.steer -= 0.03
+            print("Going left! \n")
+        elif pygame.K_d in self.pressed_keys:
+            self.steer += 0.03
+            print("Going right!\n")
```

Comment:
Do we need these print statements or are they just for debugging. If they're just for debugging, we should probably remove them.